sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/src/main/java/com/azure/messaging/eventhubs/checkpointstore/blob/BlobCheckpointStore.java

```diff
@@ -162,30 +211,33 @@ public BlobCheckpointStore(BlobContainerAsyncClient blobContainerAsyncClient) {
         String offset = checkpoint.getOffset() == null ? null : String.valueOf(checkpoint.getOffset());
         metadata.put(SEQUENCE_NUMBER, sequenceNumber);
         metadata.put(OFFSET, offset);
-        metadata.put(OWNER_ID, checkpoint.getOwnerId());
         BlobAsyncClient blobAsyncClient = blobClients.get(blobName);
-        BlobRequestConditions blobRequestConditions = new BlobRequestConditions().setIfMatch(checkpoint.getETag());
 
-        return blobAsyncClient.setMetadataWithResponse(metadata, blobRequestConditions)
-            .map(response -> response.getHeaders().get(ETAG).getValue());
+        return blobAsyncClient.setMetadata(metadata);
     }
 
-    private String getBlobPrefix(String eventHubName, String consumerGroupName) {
-        return eventHubName + BLOB_PATH_SEPARATOR + consumerGroupName;
+    private String getBlobPrefix(String fullyQualifiedNamespace, String eventHubName, String consumerGroupName,
+        String typeSuffix) {
+        return fullyQualifiedNamespace + BLOB_PATH_SEPARATOR + eventHubName + BLOB_PATH_SEPARATOR + consumerGroupName
```

Comment:
`String.join(BLOB_PATH_SEP, ...)` is easier for me to read, but not sure how optimised it is.